docs/updates/index.rst

```diff
@@ -2,6 +2,14 @@
 Updates
 #######
 
+April 28 2025
```

Comment:
When we have time I'd like to get back to how we should handle news - without an RSS feed I am not sure a static page in rsp.lsst.io is the best way but that can be a question for another day.

Reply:
Also not sure this is the best way, manually maintaining an 'updates' page... things will be missed. Would love to replace with a better option sometime.